webpack.config.analyzer.js

```diff
@@ -0,0 +1,6 @@
+import {BundleAnalyzerPlugin} from 'webpack-bundle-analyzer';
+import config from './webpack.config.dev';
+
+config.plugins.push(new BundleAnalyzerPlugin());
```

Comment:
@coryhouse, I had the analyzer config in its own webpack config just in case in the future we wanted to add other stuff to it, but I'm fine having it in the `analyzeBundle.js` script and yes it makes more sense to have it running against the prod build.